docs/core/tools/global-json.md

```diff
@@ -65,15 +140,30 @@ dotnet new globaljson --sdk-version 2.2.100
 > The matching rules are governed by the apphost, which is part of the .NET Core runtime.
```

Comment:
Great idea @tdykstra! But I don't think we have anything written anywhere about apphost. @KathleenDollard are you aware of any? If not, should we add a backlog item to explain what apphost is?

Reply:
Great catch. 

When this was written, apphost was just part of the common `dotnet.exe` code. Now, that part of the code is inserted into framework dependent shared applications, so use of the word `apphost` here is going to be confusing. And saying this is part of the runtime is confusing here because the key point is that this part is NOT side by side. Is this clear? (including the next line as well). 

```suggestion
> The matching rules are governed by `dotnet.exe` entry point, which is common across all installed .NET Core installed runtimes. The matching rules for the latest installed version of the .NET Core Runtime are used when you have multiple runtimes installed side-by-side.
```

Reply:
I think this is more clear. Thanks!

Reply:
@mairaw I may be confused here. Did you intend to take this change? Your comment sounds like yes, but I do not see this updated in the doc.

Reply:
Oh weird. Yes, I intended to take this. And I should probably remove the next line correct?

Reply:
I think this wording is fine. Super *technically*, it is governed by the highest version of hostfxr in <directory of dotnet>\host\fxr\<version>, which the dotnet entry point will load and call to  

Is it better to use a more cross-plat name than dotnet.exe (exe is windows specific), maybe just dotnet entry point?

Neither of these are very important. :)